osgar/drivers/kloubak.py

```diff
@@ -19,9 +19,9 @@
 ENC_SCALE = (33/25)*8.0/950  # TODO proper calibration (scale for large 33" wheels, old were 25")
 TURNING_ANGULAR_SPEED = math.pi/8
 
-AD_CENTER = 419.7 # K2, can be modified by config
+AD_CENTER = 531 # K2, can be modified by config
```

Comment:
why are we modifying both code and JSON files?

Reply:
The default old constants are not working any more. This is why i updated they.